tasks/main.yml

```diff
@@ -27,4 +27,4 @@
     dest: /etc/apt/preferences.d/deb_nodesource_com_node.pref
 
 - name: Install Node.js
-  apt: pkg=nodejs={{ nodejs_version }}.* state=installed update_cache=yes
+  apt: pkg=nodejs state=installed update_cache=yes
```

Comment:
An example of this in apt is.

```
apt-get install apache2=2.2.20-1ubuntu1
```

If `2.2.20-1ubuntu1` doesn't exist then it will error out.